### ISSUE TITLE
Incorrect styling of tables 

### ISSUE THREAD

**Issue by stefanalund** [backstage/backstage#180]
Today:
<img width="688" alt="Screenshot 2020-03-06 at 14 00 16" src="https://user-images.githubusercontent.com/190856/76085745-f0621a80-5fb2-11ea-8d6d-b075308debd9.png">

Should be (without the table header -- which is broken right now):
<img width="521" alt="Screenshot 2020-03-06 at 14 04 11" src="https://user-images.githubusercontent.com/190856/76085987-754d3400-5fb3-11ea-824a-9c9884bc8c53.png">
